pallets/xyk/src/lib.rs

```diff
@@ -679,66 +721,17 @@ impl<T: Trait> Module<T> {
         Ok((first_asset_amount, second_asset_amount))
     }
 
+    //TODO if pool contains key !
```

Comment:
How critical is this TODO? Is it covered by e2e?

Reply:
unfortunate artifact from dev process, don't see any reason for this todo anymore